test/test.js

```diff
@@ -12,3 +18,39 @@ describe('Utils |', function() {
     });
   });
 });
+
+describe('Utils |', function() {
```

Comment:
This is duplicated, you can have all your test inside the main `describe('Utils |', function() {`